rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -159,7 +158,7 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
   it should "throw an error if names are used against their polarity" in {
     implicit val errorLog = new ErrorLog()
     val costAccounting =
-      CostAccountingAlg.monadState(AtomicRefMonadState.of[Task, CostAccount](CostAccount.zero))
+      CostAccountingAlg.unsafe[Task](CostAccount.zero)
```

Comment:
Let's maybe have `def costAccouning() = CostAccountingAlg.unsafe[Task](CostAccount.zero)` and call it wherever we use the multiply-declared vals? WDYT?